quartz/plugins/emitters/componentResources.ts

```diff
@@ -126,6 +126,15 @@ function addGlobalPageResources(
 
       document.head.appendChild(umamiScript)
     `)
+  } else if (cfg.analytics?.provider === "goatcounter") {
+    componentResources.afterDOMLoaded.push(`
+      const goatcounterScript = document.createElement("script")
+      goatcounterScript.src = "${cfg.analytics.scriptSrc ?? "//gc.zgo.at/count.js"}"
```

Comment:
why do we use `//` prefixing the domain here?

Reply:
It will use whichever protocol the site is using already (http or https). Just prevents mixed content warnings I think. It's how they suggest to add it on their docs https://www.goatcounter.com/help/start

Reply:
cool can you just add a brief comment on that in the docs? that might be confusing for people

Reply:
Y'know, now that I'm googling around looking for a resource to point to about them, it sounds like these are outdated (and even considered an [anti-pattern](https://www.paulirish.com/2010/the-protocol-relative-url/) 😬 ). I might just change it to https since the resource is available there.